sphinx/locale/__init__.py

```diff
@@ -220,13 +223,13 @@ def gettext(message: str) -> str:
 # labels
 admonitionlabels = {
     'attention': _('Attention'),
-    'caution':   _('Caution'),
-    'danger':    _('Danger'),
-    'error':     _('Error'),
-    'hint':      _('Hint'),
+    'caution': _('Caution'),
```

Comment:
Should we keep the spaces or not? it's good to see what happens but it's not ruff-formatted. Let's decide now whether we allow spaces like that for visual purposes or not (it will affect tests a lot so let's be clear for what we should do).

(Personally, I'm for this change)

Reply:
Also, this case should be ignored by flake8 since you enabled E205 so it should be ignored by ruff as well (for now).

Reply:
Not keep the spaces, its just unecessary

Reply:
Fine to me (at least PyCharm won't complain anymore)